cdisc_rules_engine/operations/operations_factory.py

```diff
@@ -68,6 +71,7 @@ class OperationsFactory(FactoryInterface):
         "extract_metadata": ExtractMetadata,
         "get_column_order_from_dataset": DatasetColumnOrder,
         "get_column_order_from_library": LibraryColumnOrder,
+        "get_ct_attribute_values": CTAttributeValues,

```

Comment:
Maybe we could just call it "get_codelist_attributes"

Reply:
Renamed it to get_codelist_attributes.